server/services/Email_services.js

```diff
@@ -0,0 +1,37 @@
+const nodemailer = require('nodemailer');
+
+const Logger = require('./logger_services');
+
+const logger = new Logger('app');
+
+const transporter = nodemailer.createTransport({
+  service: 'GMAIL',
+  auth: {
+    user: 'kingmonkey409@gmail.com',
+    pass: 'Yos112233',
+  },
+});
+
+const SERVICE_EMAIL_ADDRESS = {
+  from: 'kingmonkey409@gmail.com',
+  to: 'ziadeyusef@gmail.com',
+};
```

Comment:
Object property is not const as you can change that at any time,
const SERVICE_EMAIL_ADDRESS = 'kingmonkey409@gmail.com';
const ADMIN_EMAIL_ADDRESS = 'ziadeyusef@gmail.com';